sklearn/datasets/descr/diabetes.rst

```diff
@@ -21,11 +21,11 @@ quantitative measure of disease progression one year after baseline.
       - sex
       - bmi     body mass index
       - bp      average blood pressure
-      - s1      tc, T-Cells (a type of white blood cells)
+      - s1      tc, total serum cholesterol
       - s2      ldl, low-density lipoproteins
       - s3      hdl, high-density lipoproteins
       - s4      tch, thyroid stimulating hormone
```

Comment:
tch is consistent with actually being total cholesterol over HDL, as my comments in issue #18940 indicate

Reply:
meaning `s4 == s1 / s3`? i tried comparing the values but they're not exactly the same. some rows differ by quite a bit. i'm hoping this is due to rounding error from the original dataset.